### ISSUE TITLE
Set tags like you set project

### ISSUE THREAD

**Issue by 0atman** [watercooler-labs/toggl-cli#83]
I can't find a way to set arbitrary tags, I was hoping for something like:

```sh
$ toggl start -p myproject -t a_tag -t or_two "description here"
```

Autotagging using git is very cool, but I'm not always working on a git project!

**Comment by 0atman** [comment]
Thank you so much!

**Comment by shantanuraj** [comment]
Yeah sorry seems like we never added tag support in the `toggl start` command.
I'll look get around to it soon.
Thanks for reporting these!

**Comment by 0atman** [comment]
Incredible, thank you! it works great!
